kalite/distributed/settings.py

```diff
@@ -37,6 +36,7 @@ def USER_FACING_PORT():
 # TODO(bcipolli): change these to "login" and "logout", respectively, if/when
 #  we migrate to a newer version of Django.  Older versions require these
 #  to be set if using the login_required decorator.
+# TODO(benjaoming): Use reverse_lazy for this sort of stuff
 LOGIN_URL = "/securesync/login/"
 LOGOUT_URL = "/securesync/logout/"
```

Comment:
Dunno, but settings.LOGIN_URL works for everything except django.contrib.admin which always redirects to the admin backend's login page.